apps/web/src/components/docs/DocsHomeModal.tsx

```diff
@@ -0,0 +1,244 @@
+import { ActionIcon, Modal, useMantineColorScheme } from '@mantine/core';
+import {
+  colors,
+  IconChecklist,
+  IconCode,
+  IconEditNote,
+  IconHandyman,
+  IconOpenInNew,
+  IconOutlineClose,
+  IconOutlineCloudUpload,
+  IconOutlineMarkEmailRead,
+  IconOutlineSwapCalls,
+  IconOutlineZoomIn,
+  IconPlayArrow,
+  IconSchool,
+  IconWidgets,
+  IconWrapText,
+  Tooltip,
+} from '@novu/design-system';
+import { css } from '../../styled-system/css';
+import { Flex, Grid, GridItem, styled } from '../../styled-system/jsx';
+import { title } from '../../styled-system/recipes';
+import { useDocsContext } from '../providers/DocsProvider';
+import { When } from '../utils/When';
+import { DocsCard } from './DocsCard';
+import { ExploreLink } from './ExploreLink';
+
+const ModalTitle = styled('h2', title);
+
+export const DocsHomeModal = ({ path, open }: { path: string; open: boolean }) => {
```

Comment:
This is not used anywhere right now but will be in the next version of the docs. I implemented this because it was not communicated to me that this was v2 so to say :)